src/store/CategoricalDisplayStore.ts

```diff
@@ -343,6 +344,7 @@ export default class CategoricalDisplayStore {
         if (filterText === '') {
             setTimeout(() => this.scrollToActiveSymbol(), 1);
         }
+        this.mainStore.state.debouncedStateChange(STATE.MARKET_SEARCH, { search_string: filterText });
```

Comment:
the purpose of this is to send statechange when user stops typing for a while right? instead of sending an state/event at every character change

Reply:
@henry-deriv yes, correct :)